src/main/java/sx/blah/discord/handle/impl/obj/Channel.java

```diff
@@ -176,18 +180,18 @@ public void addToCache(IMessage message) {
 		}
 	}
 
-	private IMessage[] requestHistory(Long before, int limit) {
+	public IMessage[] requestHistory(Long before, Long after, int limit) {
```

Comment:
-1 will at some point be a possible id but that is far in the future. I'd recommend if you are using trove collections to use `gnu.trove.impl.Constants#DEFAULT_LONG_NO_ENTRY_VALUE` which is 0 by default but can be changed by changing the system property `gnu.trove.no_entry.long`. Otherwise you can just use 0 as well